yb-voyager/cmd/exportData.go

```diff
@@ -135,9 +134,17 @@ func exportDataOffline() bool {
 	fmt.Printf("num tables to export: %d\n", len(finalTableList))
 	utils.PrintAndLog("table list for data export: %v", finalTableList)
 
+	tablesColumnList, unsupportedColumnNames := source.DB().GetColumnsWithSupportedTypes(finalTableList, useDebezium)
+	if len(unsupportedColumnNames) > 0 {
+		log.Infof("preparing column list for the data export without unsupported datatype columns: %v", unsupportedColumnNames)
+		if !utils.AskPrompt("\nThe following columns data export is unsupported:\n" + strings.Join(unsupportedColumnNames, "\n") +
+			"\nDo you want to ignore these columns and export data of tables with supported columns") {
+			utils.ErrExit("Exiting at user's request. Use `--exclude-table-list` flag to continue without these tables")
```

Comment:
change this:
`Do you want to ignore these columns and export data of tables with supported columns` 
to:
`Do you want to ignore just these columns' data and continue with export`